consensus/ibft/fork/manager.go

```diff
@@ -0,0 +1,381 @@
+package fork
```

Comment:
Please add better documentation to this package because some parts of the implementation are really not clear 🙏

Reply:
Added comments at 5f134d93a64aa2fa150f86293927cde04185ce41